dns-query/src/index.ts

```diff
@@ -0,0 +1,6 @@
+import { expose, util } from '@chainlink/ea-bootstrap'
+import { execute, inputParams } from './adapter'
+
+const NAME = 'DNS-Query'
+
+export = { NAME, inputParams, execute, ...expose(util.wrapExecute(execute)) }
```

Comment:
No need to export the `inputParams`, but you will need to export the `makeExecute` & `makeConfig` factory functions:

```suggestion
export = { NAME, makeExecute, makeConfig, ...expose(util.wrapExecute(makeExecute())) }
```